lib/src/app/data.dart

```diff
@@ -10,13 +10,22 @@ import 'package:viam_sdk/src/gen/google/protobuf/any.pb.dart';
 import 'package:viam_sdk/src/utils.dart';
 
 import '../gen/app/data/v1/data.pbgrpc.dart';
-import '../gen/app/datasync/v1/data_sync.pbgrpc.dart' hide CaptureInterval;
 import '../gen/app/dataset/v1/dataset.pbgrpc.dart';
+import '../gen/app/datasync/v1/data_sync.pbgrpc.dart' hide CaptureInterval;
 import '../gen/google/protobuf/timestamp.pb.dart';
 import '../media/image.dart';
 
 typedef DatabaseConnection = GetDatabaseConnectionResponse;
 
+class DataResponse {
+  final List<TabularData>? tabularData;
+  final List<BinaryData>? binaryData;
+  final Int64 count;
+  final String last;
+
+  DataResponse(this.tabularData, this.binaryData, this.count, this.last);
+}
+
```

Comment:
I saw online that it was standard in Dart to make a class instead of returning a tuple. Let me know if this isn't what I should do.